vmdb/app/models/miq_report/generator/sorting.rb

```diff
@@ -49,7 +49,6 @@ def build_sort_suffix_data
     rpt_sortby = self.sortby.to_miq_a
     rpt_sortby.each do |sb|
       col, sfx = sb.split("__")
-      dt = self.class.get_col_type(self.col_to_expression_col(sb))
```

Comment:
This is why I hate abbreviations...what the hell is a `dt`?  For that matter, what the hell is a `sb` or a `sfx`?  Sandbox and Sound..effects?  Oh `sfx` is suffix, but the only reason I figured that out is because down below we lucked out and called a method called `is_break_suffix?`.  If it were just spelled out to begin with there is no question or brain overhead at all.  :tired_face: